src/qt/askpassphrasedialog.cpp

```diff
@@ -129,7 +129,20 @@ void AskPassphraseDialog::accept()
             {
                 if(model->setWalletEncrypted(true, newpass1))
                 {
-                    QMessageBox::warning(this, tr("Wallet encrypted"),
+                    if (model->hdEnabled()) {
+                        QMessageBox::warning(this, tr("Wallet encrypted"),
+                                         "<qt>" +
+                                         tr("%1 will close now to finish the encryption process. "
+                                         "Remember that encrypting your wallet cannot fully protect "
+                                         "your dashs from being stolen by malware infecting your computer.").arg(tr(PACKAGE_NAME)) +
```

Comment:
Nit: might recommend changing `your dashs` to `your Dash`

Reply:
Hmm... maybe just `funds` then?

Reply:
That works too! 👍